CONTRIBUTING.md

```diff
@@ -8,6 +8,8 @@ Hi! Thanks for checking out Julia. If you have questions or problems, the [Julia
 
 * [Fork Julia](https://github.com/JuliaLang/julia/fork).
 
+* Sign up for [Travis](https://travis-ci.org/profile/) and switch on your julia fork.
```

Comment:
I would argue that most users wouldn't be doing mac specific things, I like tests to be green before making a PR, but agree appveyor won't run either.... removed!